src/components/form/form.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button, Field, FormGeneric, FormPanel } from '@mdb/flora';
+import {
+  jobFunctions,
+  institutionTypes,
+  teachingStatuses,
+} from 'src/lib/registration';
+import FormProps from './types';
+import styles from './styles';
+
+export default function Form({
+  isOpen,
+  onClose,
+}: FormProps): JSX.Element | null {
+  return isOpen ? (
+    <div data-testid="form-modal" sx={styles.FormModalWrapper}>
+      <div sx={styles.FormModal}>
+        {/* @ts-ignore */}
+        <FormPanel
+          title="Join the MongoDB Educator Community"
+          onClose={onClose}
+        >
+          <FormGeneric
+            inverse
+            customStyles={{
+              width: ['100%', '440px', null, '100%'],
+            }}
+          >
+            <Field
+              component="text-input"
+              // validators={[isRequired(), minLength(3), maxLength(10)]}
```

Comment:
Do we intentionally want to leave these here commented?

Reply:
left them commented for now, didn't know what req's would look like for form validation. Was gonna address it during integration